app/clients/sms/aws_sns.py

```diff
@@ -26,7 +26,7 @@ def get_name(self):
         return self.name
 
     @statsd(namespace="clients.sns")
-    def send_sms(self, to, content, reference, multi=True, sender=None, template_id=None, service_id=None):
+    def send_sms(self, to, content, reference, multi=True, sender=None, template_id=None, service_id=None, sending_vehicle=None):
```

Comment:
Should `sending_vehicle` be used somewhere in this method?

Reply:
No, it's only Pinpoint that can direct to shortcode or not. If it gets to SNS then it'll just try to send using any number (and if it's sent to this person in the past, try to use the same number as before). But in [send_to_providers](https://github.com/cds-snc/notification-api/blob/main/app/delivery/send_to_providers.py#L116-L124) it doesn't know what provider it has, so it gives all the arguments.

Reply:
Ah ok cool gotcha - LGTM!